recipes/libmikmod/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,25 @@
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Libmikmod is a pure c library.
I think this example can be converted to a c source.